src/hooks/useViolations.ts

```diff
@@ -4,12 +4,12 @@ import type {TransactionViolation, ViolationName} from '@src/types/onyx';
 /**
  * Names of Fields where violations can occur.
  */
-type ViolationField = 'amount' | 'billable' | 'category' | 'comment' | 'date' | 'merchant' | 'receipt' | 'tag' | 'tax';
+type MoneyRequestField = 'amount' | 'billable' | 'category' | 'comment' | 'date' | 'merchant' | 'receipt' | 'tag' | 'tax';
```

Comment:
Why this change out of curiosity? Isn't ViolationField more accurate? The comment would need to be updated too, but I kinda like ViolationField better than MoneyRequestField.

Reading the comment below "Map from Violation Names to the field where that violation can occur." and the type of `violationFields`, ViolationField fits better imo

Reply:
Any thoughts on this one?

Reply:
Sure doesn't really bother me.